scenarios/kubernetes_e2e.py

```diff
@@ -402,6 +402,8 @@ def main(args):
     cluster = cluster_name(args.cluster, os.getenv('BUILD_NUMBER', 0))
     runner_args.extend(args.kubetest_args)
 
+    # TODO(fejta): Take this value through a flag instead of env var.
+    runner_args.append('--gcs-artifacts-dir=%s' % os.environ.get('GCS_ARTIFACTS_DIR', ''))
```

Comment:
How about this: in `jobs/config.json` we set `--enable-logexporter=true` and then if that is true and `GCS_ARTIFACTS_DIR` is set then we append --kubetest-whatever-logexporter-flag=$GCS_ARTIFACTS_DIR`

Also:
* we cannot add this flag until we ship a new kubekins-e2e image that uses a kubetest build that supports the new flag.
* we should only append this flag when we want to export logs, not always